addons/compat_rf/compat_rf_realisticnames/CfgVehicles.hpp

```diff
@@ -19,36 +19,54 @@ class CfgVehicles {
         displayName = SUBCSTRING(heli_light_03_unarmed_Name);
     };
 
-    class Heli_EC_01A_base_RF;
-    class Heli_EC_01A_military_base_RF: Heli_EC_01A_base_RF {
-        displayName = SUBCSTRING(ec_01a_military_Name);
-    };
-
+    //H240 Transport, Gendarmerie/ION Transport
```

Comment:
Add spaces between the start of the comment and the first word, that way it's uniform with the rest of the file.
```suggestion
    // H240 Transport, Gendarmerie/ION Transport
```

Reply:
Added!